format/index.md

```diff
@@ -212,6 +214,26 @@ The values of `type` members **MUST** adhere to the same constraints as
 can be either plural or singular. However, the same value should be used
 consistently throughout an implementation.
 
+#### Revision <a href="#document-resource-object-revision" id="document-resource-object-revision" class="headerlink"></a>
+
+The **OPTIONAL** `revision` member is an *opaque* string identifier assigned by
+the web server to represent a specific version of a resource. If the resource
+is *updated*, a new and different `revision` is assigned. This mechanism
+enables clients or servers to determine if two resources with the same `type`
+and `id` values are the same or different versions by comparing the `revision`
+value for equality.
+
+The purpose of the `revision` member is to enable *optimistic concurrency control*
```

Comment:
Why are this and the following paragraphs normative? Are not they just clarifications of intentions?